docs/typescript/workflows.md

```diff
@@ -884,9 +884,7 @@ import PCP from '../content/what-is-a-parent-close-policy.md'
 <summary>Why ContinueAsNew is needed
```

Comment:
"Why use Continue-As-New"?

Reply:
Ya, not gonna mess with the formatting of that can of worms yet ;)